rocky/rocky/views/ooi_detail.py

```diff
@@ -59,6 +60,12 @@ def post(self, request, *args, **kwargs):
 
     def handle_page_action(self, action: str) -> bool:
         try:
+            if action == PageActions.CHANGE_CLEARANCE_LEVEL.value:
+                clearance_level = int(self.request.POST.get("level"))
+                if not self.can_raise_clearance_level(self.ooi, clearance_level):
+                    return redirect("account_detail", organization_code=self.organization.code)
+                return self.get(self.request, *self.args, **self.kwargs)
```

Comment:
1) The problem I described in https://github.com/minvws/nl-kat-coordination/pull/2561#discussion_r1514420064 has not been fixed. The flow was:

- No indemnification present or not high enough clearance level -> error message
- Did not acknowledge clearance level -> redirect to the account page.

The flow will always redirect to the account detail now, but that does not make any sense, because the user can't give themself a higher clearance level and also can't set an organization indemnification on that page.

2) This code is for ScanProfileDetailView, so should be in ScanProfileDetailView, not in OOIDetailView. Code specific to a certain view should live in that view class and not be moved to superclasses.